client/src/pages/private/ParticipantView.js

```diff
@@ -144,11 +144,14 @@ export default () => {
       case 2:
         statuses = ['rejected'];
         break;
+      case -1:
+        statuses = ['all'];
```

Comment:
Could avoid a special case in the filtering by just setting statuses to `['prospecting', 'interviewing', 'offer_made', 'rejected', 'open']`.